docs/source/installation.rst

```diff
@@ -1,16 +1,16 @@
 Installation of MuDirac
 ========================
-MuDirac used CMake as a build system, and requires a C++ compiler. In order to compile it and prepare it to be executed on a Linux, Unix, or MacOS system with a working C++ compiler installes, follow these steps:
-1. download and unpack (or :literal:`git clone`) the repository on your local system;
-2. within the main folder of the repository (the one containing the :literal:`READ.md` file), create a subfolder called :literal:`build`;
-3. within the :literal:`build` folder, run the following commands:
+MuDirac uses CMake as a build system, and requires a C++ compiler. In order to compile it and prepare it to be executed on a Linux, Unix, or MacOS system with a working C++ compiler installed, follow these steps:
+#. Download and unpack (or :literal:`git clone`) the repository on your local system.
```

Comment:
```suggestion
MuDirac uses CMake as a build system, and requires a C++ compiler. In order to compile it and prepare it to be executed on a Linux, Unix, or MacOS system with a working C++ compiler installed, follow these steps:

#. Download and unpack (or :literal:`git clone`) the repository on your local system.
```

This still doesn't format correctly as is - it's not immediately obvious from https://sublime-and-sphinx-guide.readthedocs.io/en/latest/lists.html#ordered-lists but think we also need a newline at the start of the list (as a single newline is treated as whitespace).